doc/content/verification_and_validation/val-2f.md

```diff
@@ -0,0 +1,255 @@
+
+!alert warning title=Typo in formula from [!cite](zhao2020deuterium)
+There is a typo in the expression for the deuterium recombination coefficient for clean tungsten surfaces from [!cite](zhao2020deuterium) where the minus sign in the exponential is missing, even though the data shows it should be present. Consequently, we used the corrected value in our simulations, which includes the minus sign.
+
+## Results
+
+The figures below show the comparison of the TMAP8 calculation and the experimental data during desorption. The experimental data are provided by T. Schwarz-Selinger and are available [here](https://zenodo.org/records/11085134).
+
+[!cite](dark2024modelling) assumed an infinite recombination rate. This rate can be modeled in TMAP8 using the boundary conditions described above. The trap densities used in this paper have been fitted and differ from the analytical solution in [eq:trapping_sites_density_sol]. The values used at 0.1 dpa are shown in [val-2f_trap_induced_density] and are compared to the analytical solution.
+
+!media comparison_val-2f.py
+       image_name=val-2f_trap_induced_density.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=val-2f_trap_induced_density
+       caption=Trap densities as a function of damage. The analytical solutions are represented by the solid curve, while the points indicate the fitted values used in [!cite](dark2024modelling) at 0.1 dpa.
+
+[val-2f_deuterium_desorption_inf_recombination] displays the quantities of mobile, trapped, and desorbing deuterium atoms during the desorption process. During desorption, the temperature increases from 300 K to 1000 K. The amount of deuterium trapped will decrease as the temperature rises and the various trapping energies are reached, meaning that deuterium will leave the traps, become mobile, and diffuse out. During desorption, no further implantation occurs, resulting in a decrease in the number of mobile and trapped deuterium atoms and an increase in the number of desorbed deuterium atoms.
+
+Mass conservation in [val-2f_deuterium_desorption_inf_recombination] is maintained during desorption, with a 0.01% root mean squared percentage error (RMSPE) between the initial number of mobile and trapped deuterium atoms and the total number of deuterium atoms (mobile, trapped, and desorbed).
+
+[val-2f_comparison_inf_recombination] shows a good alignment with the experimental data. With an infinite recombination rate, the amount of mobile deuterium is minimal compared to the trapped deuterium, as demonstrated in [val-2f_deuterium_desorption_inf_recombination]. Consequently, the desorbed deuterium flux is solely due to the deuterium desorbing from the traps.
+
+!media comparison_val-2f.py
+       image_name=val-2f_deuterium_desorption_inf_recombination.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=val-2f_deuterium_desorption_inf_recombination
```

Comment:
Please increase the legend size for this figure and the other similar one, the text is not easily readable.